fw-child/functions.php

```diff
@@ -251,6 +251,10 @@ function child_theme_enqueue() {
 
 		wp_enqueue_script ( 'webflow', $child_vendor_dir . 'climatedata-scroll.webflow/webflow.js', array ( 'jquery' ), null, true );
 
+		wp_enqueue_script ( 'gsap' );
+		wp_enqueue_script ( 'scrolltrigger' );
+		wp_enqueue_script ( 'scroll' );
```

Comment:
See comment above regarding scripts enqueued via Motion.page.

Reply:
See reply to comment above.